hasher-matcher-actioner/webapp/src/Settings.jsx

```diff
@@ -3,11 +3,150 @@
  */
 
 import React from 'react';
+import Tabs from 'react-bootstrap/Tabs';
+import Tab from 'react-bootstrap/Tab';
 
 export default function Settings() {
   return (
     <>
-      <h1>Settings</h1>
+      <Tabs defaultActiveKey="signals" id="setting-tabs">
+        <Tab eventKey="signals" title="Signals">
+          <h2>Datasets</h2>
+          <div className="row mt-3">
+            <div className="col-xl-4 col-lg-4 col-md-6 col-sm-6 col-xs-12 mb-4">
+              <div className="card text-center">
```

Comment:
I'm assuming this was just a first stab at UI. Commonly (and this helps with the final version), you'd create a few javascript types to contain a Dataset.

Then you would create a mock array of such dataset instances. 

Then you would iterate through the array and render the individual cards.

That speeds up development and is closer to what your final solution is.

Reply:
Because react allows strong typing of props and stronger typing of components, this will become second nature over time. Imagine if you wanted to change the color of the heading to orange, you'd have to change it in 4 places now, but if you had rendered from a javascript list, you'd change it only in one place.

Reply:
Because each my components has a slightly different number of internal components, I couldn't figure out a way to easily do this until I figure out control flow in assembling components.